mne/time_frequency/tfr.py

```diff
@@ -1267,7 +1334,13 @@ def _induced_power_mtm(data, sfreq, frequencies, time_bandwidth=4.0,
     itc : np.ndarray, shape (n_channels, n_frequencies, n_times)
         Phase locking value.
     """
-    n_epochs, n_channels, n_times = data[:, :, ::decim].shape
+    if isinstance(decim, int):
+        decim = slice(None, None, decim)
+    elif not isinstance(decim, slice):
+        raise(TypeError, '`decim` must be int or slice, got %s instead'
+                         % type(decim))
+    n_epochs, n_channels, n_times_out = data[:, :, decim].shape
+    n_times = data[:, :, ::decim.step].shape[2]
```

Comment:
That's what I thought notably because the wavelet is not decimated but since it was natively written like that, I conclude I may have missed something.
Concerning the definition, are talking of the name of it ? I have just been using the native terminology.